cmake/assimp-plain-config.cmake.in

```diff
@@ -0,0 +1,9 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/@TARGETS_EXPORT_NAME@.cmake")
```

Comment:
Do you think it is more readable with the conditional ? I personally find that two separate files will help in avoid confusion, also because I imagine that the `assimp-plain-config.cmake` variant will get more logic with time. For example, in a coming PR I want to also switch ZLIB to use the imported targets provided by the official CMake `FindZLIB`, and I don't think that mixing `find_package(ZLIB CONFIG REQUIRED)` and `find_package(ZLIB MODULE REQUIRED)` in the same file would be helpful for readability and maintainability, but this is just my opinion.

Reply:
Typically, only CMake reads these config files. In addition, the `@ASSIMP_HUNTER_ENABLED@` will be set to `ON` or `OFF` in the generated config file, in case a human wants to read it. I think it is worth consolidating everything and simplifying the config file generation into a single one.

Reply:
> 
> 
> Typically, only CMake reads these config files. In addition, the `@ASSIMP_HUNTER_ENABLED@` will be set to `ON` or `OFF` in the generated config file, in case a human wants to read it. I think it is worth consolidating everything and simplifying the config file generation into a single one.

Sorry, regarding the readability comment I was referring to the `.cmake.in` file that will be read by any contributor that needs to modify it for the future. I find that having the two separate files is more readable, but it is just a personal opinion, anything that is ok for the Assimp mantainer is ok for me.